CONTRIBUTING.md

```diff
@@ -101,12 +101,12 @@ any browser by opening the `index.html` file.
 
 ## Releases
 
-To release a new version of _GLASS_, there should be a commit titled
-`REL: glass yyyy.mm` that includes the following changes:
+To release a new version of _GLASS_, there should be a commit that includes the
+following changes:
 
-- The changelog trailers since the last release are parsed into the
-  [changelog](CHANGELOG.md) under a section titled `[yyyy.mm]  (DD Mon YYYY)`. A
-  new link to the changeset is added at the bottom of the file.
+- The commits since the last release are added to the [changelog](CHANGELOG.md)
```

Comment:
I know commits is correct, but might be worth being more specific?
```suggestion
- The sqaushed pull requests since the last release are added to the [changelog](CHANGELOG.md)
```